### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by r7l** [mailhog/MailHog#267]
Would it be possible to have a new release? The last one was 2017. The project moved a bit since then. The last release is also missing the vendor directory. Would be great to have them back in for a new release.

**Comment by greasedonkey** [comment]
What are the alternative to this project?  We are having some issue with it and with no new release coming out soon we are looking to replace it.

**Comment by r7l** [comment]
Alternative? I am actually quite happy with it so far. It might not be perfect but i am only using it for testing.

**Comment by greasedonkey** [comment]
Yes, alternatives. One issue that is a deal breaker for our QA team is that we can't display the content of an email from the search results.

See #79

**Comment by r7l** [comment]
Ok, i see. Might not be an issue for me right now but i can understand why you are looking for an alternative.

**Comment by davidkarim** [comment]
We currently use mailcatcher and are considering moving to mailhog since it does look more up to date and at least moderately maintained. Would be interested in knowing about alternatives or comments regarding mailhog's future or current state.

**Comment by r7l** [comment]
I am using it and it works perfectly for what i do with it. Unless you're running into the issues mentioned above when dealing with thousands of mails, you should be fine with this. It catches your mails and shows them on the website. It does not deal perfectly with searching the mails.

**Comment by opensourceame** [comment]
You can compile the latest build with Go. It works fine. I don't understand why the repo owner doesn't release based on the current master branch.

**Comment by malys** [comment]
Please @tyndyll  could release a new version?

**Comment by webyneter** [comment]
Any updates?

**Comment by rroblik** [comment]
@webyneter +1

**Comment by KoolPal** [comment]
Any updates on a new release?

**Comment by webyneter** [comment]
Any updates on the upcoming release?

**Comment by opensourceame** [comment]
It has now been a year since the last minor version tag.  Seems like this project is not really progressing ...

**Comment by eugeniykurasov** [comment]
@ian-kent You are member. Can you please give some comment here, thanks.

**Comment by opensourceame** [comment]
Today is exactly one more year since my last comment and still no progress. 😖

**Comment by eugeniykurasov** [comment]
Looks like they back https://github.com/mailhog/MailHog/commits/master 🎉 New commits are in master.

**Comment by tyndyll** [comment]
There is a minor release happening shortly (next few days) to incorporate some submitted PR's and create new docker images.
